cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -35,7 +37,12 @@ func CreateLocalEtcdStaticPodManifestFile(manifestDir string, cfg *kubeadmapi.Ma
 	spec := GetEtcdPodSpec(cfg)
 
 	// writes etcd StaticPod to disk
-	return staticpodutil.WriteStaticPodToDisk(kubeadmconstants.Etcd, manifestDir, spec)
+	if err := staticpodutil.WriteStaticPodToDisk(kubeadmconstants.Etcd, manifestDir, spec); err != nil {
+		return err
+	}
+
+	fmt.Printf("[etcd] Wrote Static Pod manifest for a local etcd instance to %q\n", kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.Etcd, manifestDir))
```

Comment:
So this sentence is confusing to read.  
[etcd-installation-phase] vs. [etcd]  Otherwise it reads like [etcd] is the actor when kubeadm is the actor.